sippy-ng/src/releases/PayloadStreamOverview.js

```diff
@@ -0,0 +1,170 @@
+    'release',
+    StringParam
+  )
+  const [arch = props.arch, setArch] = useQueryParam('arch', StringParam)
+  const [stream = props.stream, setStream] = useQueryParam(
+    'stream',
+    StringParam
+  )
+
+  const [fetchError, setFetchError] = React.useState('')
+  const [isLoaded, setLoaded] = React.useState(false)
+  const [streamHealth, setStreamHealth] = React.useState([])
+
+  const fetchData = () => {
+    let queryString = ''
+
+    if (props.release !== '') {
+      queryString += '&release=' + safeEncodeURIComponent(props.release)
+    }
+
+    fetch(
+      process.env.REACT_APP_API_URL +
+        '/api/releases/health?' +
+        queryString.substring(1)
+    )
+      .then((response) => {
+        if (response.status !== 200) {
+          throw new Error('server returned ' + response.status)
+        }
+        return response.json()
+      })
+      .then((json) => {
+        // Find our specific stream's health in the list:
+        for (const stream of json) {
+          if (
+            stream.architecture == props.arch &&
+            stream.stream == props.stream
+          ) {
+            console.log('Found our stream')
+            console.log(stream)
+            setStreamHealth(stream)
+          }
+        }
+        setLoaded(true)
+      })
+      .catch((error) => {
+        setFetchError('Could not retrieve tags ' + error)
+      })
+  }
+
+  useEffect(() => {
+    fetchData()
+  }, [])
+
+  if (fetchError !== '') {
+    return <Alert severity="error">Failed to load data, {fetchError}</Alert>
+  }
+
+  if (!isLoaded) {
+    return <p>Loading...</p>
+  }
+
+  return (
+    <Container>
+      <Grid item md={12}>
+        <Typography variant="h3" style={{ textAlign: 'center' }}>
+          Payload Stream Overview
+        </Typography>
+        <Grid container spacing={3} alignItems="stretch">
+          <Grid item md={3}>
+            <SummaryCard
+              key="payload-acceptance-summary-overall"
+              threshold={TEST_THRESHOLDS}
+              name="Overall"
+              success={streamHealth.phase_counts.total.accepted}
+              flakes={0}
+              caption={
+                <Fragment>
+                  <Tooltip
+                    title={`${streamHealth.phase_counts.total.accepted} runs`}
+                  >
+                    <span>
+                      {Number(
+                        (streamHealth.phase_counts.total.accepted /
+                          (streamHealth.phase_counts.total.accepted +
+                            streamHealth.phase_counts.total.rejected)) *
+                          100
+                      ).toFixed(2)}
+                      % Accepted
+                    </span>
+                  </Tooltip>
+                </Fragment>
+              }
+              fail={streamHealth.phase_counts.total.rejected}
+            />
+          </Grid>
+          <Grid item md={3}>
+            <SummaryCard
```

Comment:
Likewise needs a tooltip prop.

The other option is fixing SummaryCard so it doesn't render the Info icon.